### PR TITLE
OSDOCS-7670: adds 4.13.12 to RNs

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.13

Issue:
https://issues.redhat.com/secure/Dashboard.jspa?selectPageId=12346397#:~:text=Actions-,OSDOCS%2D7670,-Release%20note%204.13.12

Link to docs preview:
https://64545--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes#ocp-4-13-12

QE review:
- [ ] QE has approved this change.

Additional information:
cc @mjpytlak and @rohennes

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -3155,3 +3155,58 @@ $ oc adm release info 4.13.11 --pullspecs
...
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-13-12"]
+=== RHBA-2023:5011 - {product-title} 4.13.12 bug fix
+
+Issued: 2023-09-12
+
+{product-title} release 4.13.12 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:5011[RHBA-2023:5011] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:5014[RHBA-2023:5014] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
```
Comment (lahinson) [inline release_notes/ocp-4-13-release-notes.adoc:3166]: Should line 3166 be commented out?

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -3155,3 +3155,58 @@ $ oc adm release info 4.13.11 --pullspecs
...
+
+For more information about this more flexible SR-IOV network deployment during NUMA-aware pod scheduling, see xref:../networking/hardware_networks/configuring-sriov-device.adoc#nw-sriov-exclude-topology-manager_configuring-sriov-device[Exclude the SR-IOV network topology for NUMA-aware scheduling].
+
+[id="ocp-4-13-12-custom-rhcos-image-for-gcp"]
+===== Using a custom {op-system-first} image for a Google Cloud Provider cluster
+
+By default, the installation program downloads and installs the {op-system-first} image that is used to boot control plane and compute machines. With this enhancement, you can now override the default behavior by modifying the installation configuration file (install-config.yaml) to specify a custom {op-systm} image. Before you deploy the cluster, you can modify the following installation parameters:
```
Comment (lahinson) [inline release_notes/ocp-4-13-release-notes.adoc:3190]:
```suggestion
By default, the installation program downloads and installs the {op-system-first} image that is used to start control plane and compute machines. With this enhancement, you can now override the default behavior by modifying the installation configuration file (`install-config.yaml`) to specify a custom {op-system} image. Before you deploy the cluster, you can modify the following installation parameters:
```
IBM Style lists "boot" as a use-with-caution term and provides this guidance: "Although _boot_ is widely recognized among technical users, where usage permits use _start_ or _turn on_."

Should `{ocp-systm}` be `{ocp-system}`?

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -3155,3 +3155,58 @@ $ oc adm release info 4.13.11 --pullspecs
...
+
+* `controlPlane.platform.gcp.osImage.project`
+* `controlPlane.platform.gcp.osImage.name`
+* `compute.platform.gcp.osImage.project`
+* `compute.platform.gcp.osImage.name`
+* `platform.gcp.defaultMachinePlatform.osImage.project`
+* `platform.gcp.defaultMachinePlatform.osImage.name`
+
+For more information about these parameters, xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installation-configuration-parameters-additional-gcp_installing-gcp-customizations[Additional Google Cloud Platform configuration parameters].
```
Comment (lahinson) [inline release_notes/ocp-4-13-release-notes.adoc:3199]:
```suggestion
For more information about these parameters, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installation-configuration-parameters-additional-gcp_installing-gcp-customizations[Additional Google Cloud Platform configuration parameters].
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 4]
```diff
@@ -3155,3 +3155,58 @@ $ oc adm release info 4.13.11 --pullspecs
...
+
+[id="ocp-4-13-12-allocatedloabalancernodepors-network-api"]
+===== Support for allocateLoadBalancerNodePorts in Service object of Network API
+The `ServiceSpec` component in Network API under `Service` object describes the attributes that a user creates on a service. The `allocateLoadBalancerNodePorts` attribute within `ServiceSpec` is now supported in {product-title} 4.13 release. The `allocateLoadBalancerNodePorts` attribute defines whether the `NodePorts` will be automatically allocated for services with type `LoadBalancer`.
```
Comment (lahinson) [inline release_notes/ocp-4-13-release-notes.adoc:3203]:
```suggestion
The `ServiceSpec` component in Network API under the `Service` object describes the attributes that a user creates on a service. The `allocateLoadBalancerNodePorts` attribute within the `ServiceSpec` component is now supported in {product-title} 4.13. The `allocateLoadBalancerNodePorts` attribute defines whether the `NodePorts` will be automatically allocated for services of the `LoadBalancer` type.
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 5]
```diff
@@ -3155,3 +3155,58 @@ $ oc adm release info 4.13.11 --pullspecs
...
+
+[id="ocp-4-13-12-bug-fixes"]
+==== Bug fixes
+* Previously, the {product-title} router directed traffic to a route with a weight of 0 when it had only one backend. With this update, the router will not send traffic to routes with a single backend with weight 0. (link:https://issues.redhat.com/browse/OCPBUGS-17107[*OCPBUGS-17107*])
```
Comment (lahinson) [inline release_notes/ocp-4-13-release-notes.adoc:3207]:
```suggestion
* Previously, the {product-title} router directed traffic to a route with a weight of 0 when it had only one back end. With this update, the router will not send traffic to routes with a single back end with weight 0. (link:https://issues.redhat.com/browse/OCPBUGS-17107[*OCPBUGS-17107*])
```
If "backend" (one word, unhyphenated) is the accepted and common spelling in OCP, feel free to use that spelling. IBM Style lists "backend" as a use-with-caution term and provides this guidance: "In most cases, write as shown: two words. If possible, use a more specific term, such as _server_, _operating system_, or _network_. Where _backend_ is part of the established product terminology, for example in some compilers or in the Python programming language, write as one word with no hyphen."

File: release_notes/ocp-4-13-release-notes.adoc [hunk 6]
```diff
@@ -3155,3 +3155,58 @@ $ oc adm release info 4.13.11 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-13-12"]
+=== RHBA-2023:5011 - {product-title} 4.13.12 bug fix
+
+Issued: 2023-09-12
+
+{product-title} release 4.13.12 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:5011[RHBA-2023:5011] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:5014[RHBA-2023:5014] advisory.
```
Comment (lahinson) [inline release_notes/ocp-4-13-release-notes.adoc:3164]: FYI, `https://access.redhat.com/errata/RHBA-2023:5011` and `https://access.redhat.com/errata/RHBA-2023:5014` open a page not found.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 7]
```diff
@@ -3155,3 +3155,58 @@ $ oc adm release info 4.13.11 --pullspecs
...
+==== Features
+
+[id="ocp-4-13-12-networking-sriov-exclude-topology"]
+===== Exclude SR-IOV network topology for NUMA-aware scheduling
+
+With this release, you can exclude advertising the Non-Uniform Memory Access (NUMA) node for the SR-IOV network to the Topology Manager. By not advertising the NUMA node for the SR-IOV network, you can permit more flexible SR-IOV network deployments during NUMA-aware pod scheduling.
+
+For example, in some scenarios, it is a priority to maximize CPU and memory resources for a pod on a single NUMA node. By not providing a hint to the Topology Manager about the NUMA node for the pod’s SR-IOV network resource, the Topology Manager can deploy the SR-IOV network resource and the pod CPU and memory resources to different NUMA nodes. In earlier {product-title} releases, the Topology Manager attempted to place all resources on the same NUMA node only.
+
+For more information about this more flexible SR-IOV network deployment during NUMA-aware pod scheduling, see xref:../networking/hardware_networks/configuring-sriov-device.adoc#nw-sriov-exclude-topology-manager_configuring-sriov-device[Exclude the SR-IOV network topology for NUMA-aware scheduling].
```
Comment (lahinson) [inline release_notes/ocp-4-13-release-notes.adoc:3185]: FYI, the page that this xref points to doesn't mention NUMA-aware pod scheduling. Is it the right page?

**Review by lahinson — COMMENTED** [review]
@JoeAldinger Nice work! A few comments for your review, but overall, looks good.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 6 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-13-release-notes.adoc:3164]: No worries, the release hasn't shipped yet. They will work once things ship. Thanks for the review.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-13-release-notes.adoc:3166]: No that is our standard text.

**Pull request merged**